libs/components/indicators/src/lib/modules/tokens/tokens.component.spec.ts

```diff
@@ -152,6 +152,19 @@ describe('Tokens component', () => {
   });
 
   describe('events', () => {
+    it('should fire the change and rendered events when the tokens change', fakeAsync(() => {
+      const changeSpy = spyOn(component, 'onTokensChange');
+      const renderedSpy = spyOn(component, 'onTokensRendered');
+      component.publishTokens();
+      fixture.detectChanges();
+      expect(changeSpy).toHaveBeenCalled();
```

Comment:
I added the change expectation here as it didn't appear we had any other tests that were testing that event.